cellprofiler/gui/editobjectsdlg.py

```diff
@@ -301,7 +301,7 @@ def print_figure(self, *args, **kwargs):
         #
         self.orig_axes = self.figure.add_subplot(1, 1, 1)
         self.orig_axes.set_zorder(1)  # preferentially select on click.
-        self.orig_axes._adjustable = "box-forced"
+        self.orig_axes._adjustable = "box"
```

Comment:
box-forced was deprecated and got removed, this doesn't appear to change anything.